contracts/src/v0.8/dev/CanaryUpkeep.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.13;
+
+import "../interfaces/KeeperCompatibleInterface.sol";
+import "../interfaces/KeeperRegistryInterface.sol";
+import "../ConfirmedOwner.sol";
+
+/**
+ * @notice A canary upkeep which requires a different keeper to service its upkeep at an interval. This makes sure that
+ * all keepers are in a healthy state.
+ */
+contract CanaryUpkeep is KeeperCompatibleInterface, ConfirmedOwner {
+  uint256 private s_keeperIndex;
+  uint256 private s_interval = 300;
```

Comment:
you can make this constant

Reply:
Ah i see you want to have a setter for it. Why not initialise it in constructor?

Reply:
Ryan and I think that 300 seconds is a good default and we should also have a way to change it. Hence 300 and the setter. I will change it to take a param for interval too.